### PR TITLE
Added scripts v1.0 to suggested structure

### PR DESCRIPTION
**Pull request by torivara**
Chose to structure this repo with the same logic as Modules repository. This way we can version the scripts and solutions separately.

The scripts are copied from [here](https://github.com/equinor/OmniaProductTeam/tree/tasb-patching-auto-start-stop/tasb-notes/scripts). Will be the pre-, and post-scripts for Update Management auto start/stop.

### CODE REVIEW

File: Solutions/Patching/Omnia-Classic/1.0/turnoffvms/README.MD [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Intended usage
+
+This script is intended to be run as a part of Update Management Pre/Post scripts.
+
+## Requirements
+
+It requires:
+
+- A valid RunAsAccount.
```
Comment (MariusStorhaug) [inline Solutions/Patching/Omnia-Classic/1.0/turnoffvms/README.MD:11]: What permissions does it require? Could be a good idea to document.

File: Solutions/Patching/Omnia-Classic/1.0/turnonvms/README.MD [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Intended usage
+
+This script is intended to be run as a part of Update Management Pre/Post scripts.
+
+## Requirements
+
+It requires:
+
+- A valid RunAsAccount.
```
Comment (MariusStorhaug) [inline Solutions/Patching/Omnia-Classic/1.0/turnonvms/README.MD:11]: What permissions does it require? Could be a good idea to document.

File: Solutions/Patching/Omnia-Standalone/1.0/turnoffvms/README.MD [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Intended usage
+
+This script is intended to be run as a part of Update Management Pre/Post scripts.
+
+## Requirements
+
+It requires:
+
+- A valid RunAsAccount.
```
Comment (MariusStorhaug) [inline Solutions/Patching/Omnia-Standalone/1.0/turnoffvms/README.MD:11]: What permissions does it require? Could be a good idea to document.

File: Solutions/Patching/Omnia-Standalone/1.0/turnonvms/README.MD [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Intended usage
+
+This script is intended to be run as a part of Update Management Pre/Post scripts.
+
+## Requirements
+
+It requires:
+
+- A valid RunAsAccount.
```
Comment (MariusStorhaug) [inline Solutions/Patching/Omnia-Standalone/1.0/turnonvms/README.MD:11]: What permissions does it require? Could be a good idea to document.

**Review by MariusStorhaug — CHANGES_REQUESTED** [review]
Thinking the folder structure should be decoupled with the services that are using it, but not sure. The way you suggest, you get duplication of code, but you get linking between service that is using the code, i.e. Patching Classic v1 and Standalone v1.

I would suggest decoupling it from the folder structure and have something like:
- Scripts/
  - <nameOfScript>/ (+ version folder?)
    - readme.md
    - <nameOfScript>.ps1|psm1 (etc)

With this you have less duplication of code, but we will have to find linking between dependencies elsewhere.

File: Solutions/Patching/Omnia-Classic/1.0/turnoffvms/README.MD (hunk 1 above)
Comment (torivara) [reply Solutions/Patching/Omnia-Classic/1.0/turnoffvms/README.MD]: Permissions needed added to readme. Readme moved to suggested new location.

**Pull request merged**